### ISSUE TITLE
4.6.2v and Lua Error. Wrath Classic 

### ISSUE THREAD

**Issue by sth666** [NephMakes/NeedToKnow#5]
**Client Version : 3.4.1 Wrath Classic** 
**Hello there,** 
Before Latest  Update (**4.6.2**) everything works well, but when i updated to version 4.6.2,  lua error start showing on BugSack  after each login in game, i try already remove all folder NeedtoKnow from Addons , and reinstall , but still nothing, as well i tryed remove my NTK profile in game / make a copy, but lua error is still showing up. 

```
2x NeedToKnow/Profile.lua:270: attempt to index field 'profiles' (a nil value)
[string "@NeedToKnow/Profile.lua"]:270: in function `UpdateActiveProfile'
[string "@NeedToKnow/ExecutiveFrame.lua"]:76: in function `f'
[string "@NeedToKnow/ExecutiveFrame.lua"]:24: in function <NeedToKnow/ExecutiveFrame.lua:21>

Locals:
self = <table> {
 SetProfileToAccount = <function> defined @NeedToKnow/Profile.lua:100
 GetBarGroup = <function> defined @NeedToKnow/BarGroup.lua:19
 UpdateActiveProfile = <function> defined @NeedToKnow/Profile.lua:262
 DeepCopy = <function> defined @NeedToKnow/Settings.lua:180
 FindAura = <table> {
 }
 GetBarGroupSettings = <function> defined @NeedToKnow/Settings.lua:44
 CopyProfile = <function> defined @NeedToKnow/Profile.lua:72
 UpdateBarGroups = <function> defined @NeedToKnow/BarGroup.lua:28
 GetNewProfileName = <function> defined @NeedToKnow/Profile.lua:48
 DeleteProfile = <function> defined @NeedToKnow/Profile.lua:130
 ProfilePanel = InterfaceOptionsNeedToKnowProfilePanel {
 }
 SetProfileToCharacter = <function> defined @NeedToKnow/Profile.lua:107
 ToggleLockUnlock = <function> defined @NeedToKnow/NeedToKnow.lua:74
 RemoveDefaultSettings = <function> defined @NeedToKnow/Settings.lua:108
 OptionSlider = <table> {
 }
 IsProfileNameAvailable = <function> defined @NeedToKnow/Profile.lua:59
 String = <table> {
 }
 m_last_cast_head = <table> {
 }
 SetProfileForSpec = <function> defined @NeedToKnow/Profile.lua:222
 Update = <function> defined @NeedToKnow/NeedToKnow.lua:54
 GetActiveProfile = <function> defined @NeedToKnow/Profile.lua:231
 m_last_cast = <table> {
 }
 Lock = <function> defined @NeedToKnow/NeedToKnow.lua:59
 ResizeButton = <table> {
 }
 ResetCharacterSettings = <function> defined @NeedToKnow/Settings.lua:39
 GetProfileForSpec = <function> defined @NeedToKnow/Profile.lua:226
 m_last_cast_tail = <table> {
 }
 CompressProfileSettings = <function> defined @NeedToKnow/Settings.lua:58
 BarMenu = <table> {
 }
 UnregisterSpellcastSent = <function> defined @NeedToKnow/ExecutiveFrame.lua:107
 UpdateBarGroup = <function> defined @NeedToKnow/BarGroup.lua:34
 Bar = <table> {
 }
 GetProfileByName = <function> defined @NeedToKnow/Profile.lua:236
 RenameProfile = <function> defined @NeedToKnow/Profile.lua:92
 barGroups = <table> {
 }
 GetProfileCopyName = <function> defined @NeedToKnow/Profile.lua:81
 accountSettings = <table> {
 }
 ImportBarSettingsFromString = <function> defined @NeedToKnow/ImportExport.lua:259
 LoadSavedVariables = <function> defined @NeedToKnow/Settings.lua:18
 ResetAccountSettings = <function> defined @NeedToKnow/Settings.lua:34
 ExecutiveFrame = NeedToKnow_ExecutiveFrame {
 }
 GetNewProfileKey = <function> defined @NeedToKnow/Profile.lua:35
 UpdateGroup = <function> defined @NeedToKnow/BarGroup.lua:38
 BarGroup = <table> {
 }
 GetBarSettings = <function> defined @NeedToKnow/Settings.lua:53
 totem_drops = <table> {
 }
 version = "4.6.2"
 Cooldown = <table> {
 }
 LSM = <table> {
 }
 CreateBlankProfile = <function> defined @NeedToKnow/Profile.lua:20
 UncompressProfileSettings = <function> defined @NeedToKnow/Settings.lua:74
 m_last_guid = <table> {
 }
 SlashCommand = <function> defined @NeedToKnow/SlashCommand.lua:12
 Unlock = <function> defined @NeedToKnow/NeedToKnow.lua:67
 characterSettings = <table> {
 }
 GetGroupSettings = <function> defined @NeedToKnow/Settings.lua:48
 MAX_BARGROUPS = 4
 MakeBarGroups = <function> defined @NeedToKnow/BarGroup.lua:12
 LoadProfiles = <function> defined @NeedToKnow/Profile.lua:140
 AddSlashCommand = <function> defined @NeedToKnow/SlashCommand.lua:6
 AppearancePanel = InterfaceOptionsNeedToKnowAppearancePanel {
 }
 RegisterSpellcastSent = <function> defined @NeedToKnow/ExecutiveFrame.lua:97
 OptionsPa
```

**Comment by sth666** [comment]
**Lua Error for sure is ending at  " OptionsPa " later nothing is there.**

**Comment by NephMakes** [comment]
Fixed in v4.6.3

**Comment by NephMakes** [comment]
Thanks for the report
